engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -6,6 +6,8 @@ import java.util.{Date, UUID}
 import javax.sql.rowset.serial.SerialClob
 
 import _root_.slick.backend.DatabaseConfig
+import _root_.slick.dbio
+import _root_.slick.dbio.Effect.Read
 import _root_.slick.driver.JdbcProfile
```

Comment:
Can somebody explain what these `_root_` imports are?  I'm not sure I've ever seen something like this before